src/Clock/Clock.php

```diff
@@ -14,4 +14,8 @@ public function now(): DateTimeImmutable;
     public function time(): int;
 
     public function sleep(int $seconds): void;
+
+    public function utime(): int;
+
+    public function usleep(int $microseconds): void;
```

Comment:
This feels like an implementation detail to me. What's the harm in just updating the implementation to make `time`/`sleep` microseconds?

Reply:
I agree with that, I have a better idea, will adjust tomorrow